markdown/developers/tutorials/dsl2_modules_tutorial.md

```diff
@@ -61,37 +61,106 @@ In order to create a new module, it is best to branch the code into a recognisab
 Using [nf-core/tools](https://github.com/nf-core/tools) it is very easy to create a new module. In our example, we change directory into the repository (_modules_) and we type
 
 ```bash
-nf-core modules create fgbio/fastqtobam
+nf-core modules create fgbio/demofastqtobam
 ```
 
 - The first word indicates the tool (i.e. the software or suite)
 - We separate the keys by a forward slash (`/`)
 - The second word indicates the function of the tool we are creating a module for
```

Comment:
same, here I would change to 

-> - `The second word indicates the function of the tool we are creating a module for, in this case "demofastqtobam"`